Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/ProjectsRepository/ProjectRepository.cs

```diff
@@ -41,194 +42,198 @@ public class ProjectRepository : IProjectRepository
         #endregion
 
         /// <summary>
-        /// Get All Projects list from the database
+        /// Method to return list of projects 
         /// </summary>
-        /// <returns></returns>List of Projects
+        /// <returns>list of projects</returns>
         public async Task<IEnumerable<ProjectAc>> GetAllProjects()
         {
             var projects = await _projectDataRepository.GetAll().ToListAsync();
-            var projectAcs = new List<ProjectAc>();
+            var projectAcList = new List<ProjectAc>();
 
             projects.ForEach(project =>
```

Comment:
why have you used ForEach lamda method where as we usually use foreach loop? This breaks conventions